devlab/tests/config.py

```diff
@@ -91,17 +91,34 @@
      'is_public': True},
     {'name': 'image2', 'copy_from': 'http://download.cirros-cloud.net/0.3.3/ci'
                                     'rros-0.3.3-x86_64-disk.img',
+     'container_format': 'bare', 'disk_format': 'qcow2', 'is_public': False},
+    {'name': 'image3', 'copy_from': 'http://download.cirros-cloud.net/0.3.3/ci'
```

Comment:
Super minor, but could you please move the line continuation so that 'cirros' could be read as a single word? This would greatly improve readability.